dev/content/pages/About.md

```diff
@@ -105,15 +105,18 @@ save_as: index.html
 <!-- Left -->
 <div class="about-box">
 <h1 style="margin-top:0; text-align: left;">About Me</h1>
-    <p>I am a data scientist at Google where I use data to inform decisions, although all my opinions here are my own. I have previously worked at Sweetgreen, designing systems that would benefit team members and communities through sustainable and healthy food, and SpaceX, creating tools that would ultimately launch rocket ships. I feel fortunate to have worked with amazing teams throughout my career.</p>
-    <p>There are 3 things I strive to be:</p>
+    <p>I am a data scientist at Google. My focus is supporting strategic decision making with data, although all my opinions here are my own. Previously, I worked at Sweetgreen and SpaceX, doing the same line of work as now but on different applications, which is using data to inform staffing strategy and rocket launch cadence.</p>
+    
+    <p>To become a better "decision" scientist, I'm currently focusing on three topics:</p>
     <ol>
-        <li>An expert computational mathematician</li>
-        <li>An effective community member and leader</li>
-        <li>A knowledge contributor</li>
+        <li>Organizational influence and leadership</li>
+        <li>Advanced applied mathematics, e.g. Causal Inference, Decision Theory</li>
+        <li>Something else TBD</li>
```

Comment:
Ok I didn't mean for you to get rid of it completely D: The business-related idea was a good one but not sure the best way to phrase it. 

Some ideas I can think of are (still pretty raw):
* Approaches to problem solving for businesses
* Combining multiple perspectives for problem solving

Reply:
More spitballing, deep expertise in the "whatever your company does?"